lib/redux/reducers.dart

```diff
@@ -15,7 +15,7 @@ AppState appReducer(AppState state, dynamic action) =>
 final globalReducer = combineReducers<GlobalState>([
   TypedReducer<GlobalState, SwitchTabAction>(_switchTabAction),
   TypedReducer<GlobalState, RehydrateSuccessAction>(_rehydrateSuccessAction),
-  TypedReducer<GlobalState, LoginAction>(_loginAction),
+  TypedReducer<GlobalState, LoginSuccessAction>(_loginAction),
```

Comment:
We should also rename the handling method from `_loginAction ` to `_loginSuccessAction `

Reply:
I already added this to the app privacy in itunesconnect.